README.md

```diff
@@ -142,8 +144,8 @@ A: No, it is a completely free aircraft, open-source.
 
 **Q: How do we report bugs?**
 
-A: Report bugs to us in the [Discord server](https://discord.gg/flybywire), under the #help channel, or by creating a GitHub issue. Just make sure to search for existing issues first before creating a new one.
+A: You can report bugs to us via the [#help channel](https://discord.gg/snueqJjDUN). Addtionally, you can start/create a GitHub issue (please make sure your issue is not already known and the same as another user). 

```

Comment:
Why remove the Discord mention? How about "the #help channel on our Discord server".

Reply:
I'd also change the wording 

```suggestion
A: You can report bugs to us via the [#help channel](https://discord.gg/snueqJjDUN). Addtionally, you can start/create a GitHub issue (please make sure your issue doesn't already exist). 

```